macros/macros_docs.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
```

Comment:
Super minor, can we rename this file to be `quickbooks_macros.sql`. This will just be consistent with the other speciality yml files we have in other packages.

Reply:
Updated!